Artifacts/windows-powershellcore/installPowerShellCore.ps1

```diff
@@ -0,0 +1,79 @@
+﻿param
```

Comment:
Please follow structure of [`artifact.ps1`](https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/windows-test-paramtypes/artifact.ps1) in artifact [`Test Parameter Types`](https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-test-paramtypes) to ensure all errors bubble up correctly. Observe the use of `trap` and `try`/`finally`.